pubspec.yaml

```diff
@@ -1,18 +1,18 @@
 name: source_gen
-version: 0.5.4+3
+version: 0.5.4+4
 author: Dart Team <misc@dartlang.org>
 description: Automatic sourcecode generation for Dart
 homepage: https://github.com/dart-lang/source_gen
 environment:
   sdk: '>=1.12.0 <2.0.0'
```

Comment:
I will end up doing that with https://github.com/dart-lang/source_gen/pull/142

Would prefer for it to happen there, because that is the commit which makes the code need 1.21 rather than just deps (which pub will be checking for us)